### ISSUE TITLE
Fix spelling of 'used' in README.md and src/Command/UnusedCommand.php

### ISSUE THREAD

**Issue by rcampbel3** [icanhazstring/composer-unused#14]
**Describe the bug**
Thanks for a really neat tool! As I was generating reports, I noticed that 
used is spelled 'usued' in README.md and UnusedCommand.php

```
ross@ubuntu1804:~/project/composer-unused $ ag usued .
README.md
61:Found 13 usued and 0 unused packages

src/Command/UnusedCommand.php
97:                'Found <fg=green>%d usued</> and <fg=red>%d unused</> packages',
```

**Comment by icanhazstring** [comment]
Hi @rcampbel3 nice catch. If you want just file a PR. I am happy to merge it 😊

**Comment by machour** [comment]
Opened #16 to fix the typo before I saw your issue @rcampbel3, hope you won't mind 😅

**Comment by icanhazstring** [comment]
Fix with #16. Thank you both tho 😉
